src/sentry/static/sentry/app/views/releases/releaseOverview.jsx

```diff
@@ -285,15 +286,20 @@ const ReleaseOverview = createReactClass({
               {!deploys.length
                 ? this.renderEmpty()
                 : deploys.map(deploy => {
-                    let query = encodeURIComponent(
-                      `environment:${deploy.environment} release:${version}`
-                    );
+                    let href = `/${orgId}/${projectId}/?query=release:${version}&environment=${deploy.environment}`;
```

Comment:
don't you still need to use encodeURI on this branch for whitespace and stuff?
It's hard for me to see the difference between these two branches actually

Reply:
👍 for encoding. The difference is that in the new environments world, environment is no longer part of "query", it is it's own separate parameter.